.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I see you're missing `test.md` in your .gitignore file. Please fix this since is part of the requirements for this project.

> - A test.md file with the text "This file should be ignored by git".
> - Make sure that the last file is actually ignored by git and is not present in your GitHub repository.